DigitalLearningSolutions.Web/Views/Shared/Components/CentreInfo/Default.cshtml

```diff
@@ -0,0 +1,7 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@model CentreInfoViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/shared/centreInfo.css")" asp-append-version="true">
+
+<p class="nhsuk-body-l">@Model.BannerText</p>
+<vc:action-link asp-controller="FindYourCentre" asp-action="Index"  asp-all-route-data="@Model.Routes" link-text="More information" />

```

Comment:
Super minor, but could we name this "CentreContactInfo" in various places to be consistent with the "BannerText" rename in the admin UI, which will now be "Centre contact info". 🙏

Reply:
Also, NIP, but the asp-all-route-data has two spaces in front of it 👀